rdr_service/dao/participant_summary_dao.py

```diff
@@ -830,7 +837,7 @@ def update_enrollment_status(self, summary: ParticipantSummary, session,
             wgs_sequencing_time=wgs_sequencing_time,
             exposures_authored_time=first_exposures_response_time,
             is_pediatric_participant=summary.isPediatric,
-            has_linked_guardian_accounts=(summary.relatedParticipants and len(summary.relatedParticipants) > 0)
+            has_linked_guardian_accounts=(summary.guardianParticipants and len(summary.guardianParticipants) > 0)
```

Comment:
:nit Should be no need for the extra operator the list should evaluate to be greater than 0.